examples/visual-tests/amp-story/amp-story-page-attachment.js

```diff
@@ -224,4 +224,38 @@ module.exports = {
         '.i-amphtml-story-page-attachment-domain-label',
     ]);
   },
+
+  'form presence - the keyboard should close upon attachment dismissal': async (
```

Comment:
Since soft-keyboard is system level and dismissal is a side effect of blurring, I wonder if this test would be better described as something like "The input field should blur on close".
We could mention that it's for the soft keyboard in the last comments below.